qemu/tests/win_nics_teaming.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import time
+import os
+import random
+
+from avocado.utils import process
+from virttest import utils_net
+from virttest import utils_misc
+
+
+def run(test, params, env):
+    """
+    Nic teaming test in guest.
+
+    1) Start guest with four nic devices.
+    2) Setup Team in guest.
+    3) Execute file transfer from host to guest.
+    4) Repeatedly set enable/disable interfaces by 'netsh interface set'
+    5) Execute file transfer from guest to host.
+
+    :param test: Kvm test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    def copy_file_from_host(host_path, guest_path, vm):
+        # transfer data
+        logging.info("Transfer data from host to %s" % vm.name)
+        vm.copy_files_to(host_path, guest_path)
+
+    def copy_file_from_guest(guest_path, host_path, vm):
+        logging.info("Transfer data from %s to host" % vm.name)
+        vm.copy_files_from(guest_path, host_path)
+
+    tmp_dir = params["tmp_dir"]
+    filesize = int(params.get("filesize"))
+    dd_cmd = params["dd_cmd"]
+    delete_cmd = params["delete_cmd"]
+    login_timeout = int(params.get("login_timeout", 1200))
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session_serial = vm.wait_for_serial_login(timeout=login_timeout)
+    ifnames = []
+    for i in range(0, 4):
```

Comment:
It should depend on the length of nics, not hard code.

Reply:
I also have doubts here, I didn't understand what you mean?

Reply:
The length should be the count of NIC cards.

Reply:
I have solved above problem,please help me review it again,thanks.